{"name": "test_login_fail[data0]", "status": "passed", "attachments": [{"name": "log", "source": "53e41698-54fe-4239-8e00-8348e84f996a-attachment.txt", "type": "text/plain"}], "parameters": [{"name": "data", "value": "{'id': 1, 'title': '5位验证码', 'url': 'http://192.168.3.157:8069/dongwang-app/app/user/login', 'json': {'type': '1', 'flashToken': '', 'deviceId': 'aaa', 'device': '1', 'smsCode': '12312', 'phone': '113157121093'}, 'expected': {'status_code': 500, 'message': '验证码输入错误,请重新输入'}}"}], "start": 1614590581237, "stop": 1614590581274, "uuid": "145cbe20-09ec-473f-87a7-24be1968f5c3", "historyId": "658c31268613a37e166d88d3437a32fc", "testCaseId": "a67639c7d108120b92ca3b258a406cc7", "fullName": "testcase.test_login.TestLogin#test_login_fail", "labels": [{"name": "story", "value": "错误验证码"}, {"name": "feature", "value": "登录模块"}, {"name": "parentSuite", "value": "testcase"}, {"name": "suite", "value": "test_login"}, {"name": "subSuite", "value": "TestLogin"}, {"name": "host", "value": "SC-201907251023"}, {"name": "thread", "value": "3528-MainThread"}, {"name": "framework", "value": "pytest"}, {"name": "language", "value": "cpython3"}, {"name": "package", "value": "testcase.test_login"}]}